### PR TITLE
dove run <call> or dove tx <call>: hints changed

### PR DESCRIPTION
**Pull request by vladimirovmm**

### CODE REVIEW

File: dove/tests/test_cmd_dove_tx.rs [hunk 1]
```diff
@@ -42,9 +42,9 @@ fn test_cmd_dove_tx_without_arguments() {
 }
 
 /// $ dove tx -n sdemo_4 -a 16 -t u8
-/// $ dove tx 'sdemo_4()' -a 16 -t u8
-/// $ dove tx 'sdemo_4(16) -t u8'
-/// $ dove tx 'sdemo_4<u8>(16)'
+/// $ dove tx sdemo_4() -a 16 -t u8
+/// $ dove tx sdemo_4(16) -t u8
+/// $ dove tx sdemo_4<u8>(16)
```
Comment (mkurnikov) [inline dove/tests/test_cmd_dove_tx.rs:47]: @vladimirovmm Could you add test for multiple parameters of the script? I'm not sure it's going to work without quotes

**Review by mkurnikov — CHANGES_REQUESTED** [review]

File: dove/src/transaction.rs [hunk 2]
```diff
@@ -537,89 +545,110 @@ fn parse_text_into_scripts(content: &str) -> Result<Vec<(String, String)>, Error
...
     let mut arguments = vec![];
-
-    lexer.advance().map_err(map_err)?;
+    lexer.advance().map_err(|_| anyhow!("{}", &error_message))?;
     while lexer.peek() != Tok::RParen {
         if lexer.peek() == Tok::EOF {
-            anyhow::bail!("Invalid call script format.\
-                 Invalid arguments format.. Use pattern \
-                 'script_name<comma separated type parameters>(comma separated parameters WITHOUT signers)'");
+            anyhow::bail!("{}", &error_message);
```
Comment (fzzr-) [inline dove/src/transaction.rs:621]: I'm not sure but is there `"Invalid arguments format"` message missed?

File: dove/src/transaction.rs [hunk 3]
```diff
@@ -537,89 +545,110 @@ fn parse_text_into_scripts(content: &str) -> Result<Vec<(String, String)>, Error
...
-                lexer.advance().map_err(map_err)?;
+                lexer.advance().map_err(|_| anyhow!("{}", &error_message))?;
             }
         }
         arguments.push(token);
         if !sw && lexer.peek() == Tok::RParen {
             break;
         }
-        lexer.advance().map_err(map_err)?;
+        lexer.advance().map_err(|_| anyhow!("{}", &error_message))?;
```
Comment (fzzr-) [inline dove/src/transaction.rs:651]: Why don't we just use single closure instead of various four?

**Review by fzzr- — CHANGES_REQUESTED** [review]
Increment of Patch part of Dove version is missed.

**Review by fzzr- — APPROVED** [review]

**Pull request merged**